src/auth/oauth2client.ts

```diff
@@ -68,6 +68,12 @@ export enum CertificateFormat {
   JWK = 'JWK',
 }
 
+export enum ClientAuthentication {
```

Comment:
Can you add a comment explaining what this enum is for?

Reply:
Added